commit 9947f2df19a6c5b5706ed3b002199dbb9fef17b1
Author: Dmitriy Fedchenko <xfedch@gmail.com>
Date:   Mon Jun 6 12:46:14 2022 -0300

    socket: Fix mistyped define statement in socket/sys/socket.h (BZ #29225)
    
    (cherry picked from commit 999835533bc60fbd0b0b65d2412a6742e5a54b9d)

diff --git a/socket/sys/socket.h b/socket/sys/socket.h
index bd14e7e3a5008ec5..5765dc1f0dedd380 100644
--- a/socket/sys/socket.h
+++ b/socket/sys/socket.h
@@ -181,7 +181,7 @@ extern ssize_t __REDIRECT (sendmsg, (int __fd, const struct msghdr *__message,
 # else
 extern ssize_t __sendmsg64 (int __fd, const struct msghdr *__message,
 			    int __flags);
-#  defien sendmsg __sendmsg64
+#  define sendmsg __sendmsg64
 # endif
 #endif
 
